docs/tutorials/bson-v4.txt

```diff
@@ -549,9 +549,9 @@ follows:
 
 .. note::
 
-  In BSON 5.0 ``BSON::Decimal128`` will be deserialized into ``BigDecimal`` by
+  In BSON 5.0, ``BSON::Decimal128`` will be deserialized into ``BigDecimal`` by
   default. In order to have ``BSON::Decimal128`` be deserialized into
```

Comment:
I would say "In order to have Decimal128 values in BSON documents deserialized into ..."